docs/source/model_design/vivarium_model_components/impairments/index.rst

```diff
@@ -33,17 +33,224 @@ Modeling Impairments
+
+An easy example to grasp is blindness. There is a lot of data available about blindness, but 
+it can be caused by a lot of different things. Therefore, it is difficult to place in the 
+cause hierarchy, but we want to include all the rich data we have. Hence, it is an 
+impairment. 
+
+Modeling impairments can be important for a simulation depending on the research questions. 
+Continuing the blindness example above, if you wanted to include an intervention that 
+targets blindness generally, you would ideally want to include blindness, but 
+wouldn't necessarily want to include all of the possible causes for blindness. 
+Hence the need to model the impairment instead of all causes of the impairment. Including 
+all of the causes would unnecessarily complicate the model and include information 
+not relevant to our research question. 
+
+On the other hand, if we were investigating an intervention that targets diabetes, 
+we might not need to include blindness as an impairment. Instead we could just track 
+the sequela "blindness due to diabetic mellitus" since we are only concerned about 
+blindness due to a single cause. 
+
+It is important to think through your research question and what is being affected 
+by your interventions to determine if including an impairment is needed. 
 
 Comparing and Contrasting Impairment and Cause Models
 -----------------------------------------------------
 
-- Most of the modeling is very similar and we can utilize the cause model page 
-- What is not applicable from that page? (e.g., certain data sources) 
+In general, modeling impairments is similar to modeling causes - they are 
+both medical conditions simulants can experience. Therefore, most of 
+the :ref:`cause modeling page <models_cause>` is still applicable and 
+should generally be used for modeling impairments. 
+
+However, there are a few important differences between modeling causes 
+and impairments that we will cover here. These differences largely 
+pertain to data sources and availability. 
+
+Disease Models
+++++++++++++++
+
+The GBD data availability differs by impairment significantly. For 
+example, heart failure has an incidence, prevalence, excess 
+mortality rate, and disability weights. Therefore, you could create a heart 
+failure model more of less exactly as you might make a cause model. 
+
+On the other hand, anemia does NOT have an incidence or any mortality 
+associated with it. It does have a prevalence and disability weight. 
+Therefore, other methods might be needed in order to have a dynamic 
+anemia model. You can see one example of this based on simulant's hemoglobin 
+levels in the :ref:`hemoglobin, anemia and iron deficiency model <2019_hemoglobin_anemia_and_iron_deficiency>`.
+
+Risk Models
++++++++++++
+
+GBD does NOT include relative risks or PAFs connecting risk factors 
+to impairments. Therefore, if you need to model the impact of a risk 
+factor to an impairment you will need to use literature values for 
```

Comment:
It seems like you _could_ in theory use GBD RRs if you assume that the risk factor has the same effect on all sequelae within a given cause. (I think this is basically what GBD assumes?) But, I can definitely imagine situations where that is far from the truth.

Reply:
Hmm interesting. Are you thinking that we would assume that the RR for say BMI -> Ischemic Heart Disease == BMI -> Heart Failure from IHD? And then even possibly using this or some average across the causes to say the RR for BMI -> HF in general is X. 

Or were you thinking we could take the general impairment bucket (assuming there is one) like blindness from other causes and just use that for the whole impairment? 

This is an interesting idea either way. For some impairments like HF something like this could work well. But, you could also imagine that blindness due to diabetes is tied to say, sugar consumption but blindness due to meningitis is not tied to sugar at all. So this might not work as well. 

Maybe we can discuss this more in our next meeting - it seems like the RR might depend on how dispirit the causes of an impairment are.

Reply:
Yes, I was thinking that you would take the _weighted average_ of the RRs of BMI -> Ischemic Heart Disease and BMI -> Hypertensive Heart Disease, weighted by the prevalence of Heart Failure from IHD and Heart Failure from HHD respectively (pretending there are only two causes -- it would actually be more).

And I agree I think this may depend on the impairment. The diabetes example is a good one (it would be strange for sugar consumption to be a risk factor for blindness due to meningitis, especially if we were reporting that outcome separately).

I was imagining something even weirder, though I don't know of an example: imagine that there was a risk factor that (in real life) put someone at risk for mild disease but not for severe disease. If only severe disease is included in the impairment, with this approach we would say that risk factor acts on the impairment, even if they are intuitively not linked at all.